specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-06-01-preview/machineLearningServices.json

```diff
@@ -1270,6 +1270,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/updateIdleShutdownSetting": {
```

Comment:
>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/updateIdleShutdownSetting [](http://example.com/codeflow?start=5&length=186)

Note that adding an API requires a new API version. If you truly want to add this without adding an API version, you need to get approval for a breaking change. Ref: [https://aka.ms/azbreakingchangespolicy](https://aka.ms/azbreakingchangespolicy).